lib/scsi_defs.h

```diff
@@ -90,5 +95,63 @@ namespace scsi_defs {
     ControlByte control_byte;
   } ABSL_ATTRIBUTE_PACKED; static_assert(sizeof(TestUnitReadyCommand) == kTestUnitReadyCommandSize);
 
+  // SCSI Reference Manual Table 76 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct PersistentReserveInCommand {
+    OpCode op_code = OpCode::kPersistentReserveIn;
+    uint8_t reserved_1 : 3;
+    uint8_t service_action : 5;
+    uint64_t reserved_2 : 40;
+    uint16_t allocation_length;
+    ControlByte control_byte;
+  } ABSL_ATTRIBUTE_PACKED; static_assert(sizeof(PersistentReserveInCommand) == kPersistentReserveInCommandSize);
```

Comment:
prefer having `static_assert` in a separate line.

Reply:
done!